# use of try..except to handle errors
try:
    int("a")
except:
    print(f"Sorry! we dont do that here")
